src/server/cloak/index.js

```diff
@@ -90,7 +89,16 @@ module.exports = (function() {
         delete config.express;
       }
 
-      io.set('log level', config.logLevel);
+      // Cloak defaults Socket.IO log level to 1
+      io.set('log level', 1);
```

Comment:
I'm OK with the breaking change, I follow the practice of using minor version numbers for breaking changes if the major version number is zero (since it doesn't make sense to release a 1.0.0 for this)